### PR TITLE
Chore: Remove liveTimer reference from testdata

### PR DESCRIPTION
**Pull request by andresmgot**
**What is this feature?**

No functional change here since `liveTimer.ok` is an observable so the condition removed resolved always to `true`.

**Which issue(s) does this PR fix?**:

Related to https://github.com/grafana/grafana/issues/71329

**Special notes for your reviewer:**

Please check that:
- [X] It works as expected from a user's perspective.
- [X] If this is a pre-GA feature, it is behind a feature toggle.
- [X] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/plugins/datasource/testdata/runStreams.ts [hunk 1]
```diff
@@ -104,18 +102,13 @@ export function runSignalStream(
...
-          data: [frame],
-          key: streamId,
-          state: LoadingState.Streaming,
-        });
-        lastSent = liveTimer.lastUpdate;
-      }
-
+      lastSent = Date.now();
+      addNextRow(lastSent);
+      subscriber.next({
```
Comment (academo) [inline public/app/plugins/datasource/testdata/runStreams.ts:107]: I understand that `liveTimer.ok` was returning always true but I wonder if this original condition is relevant and it was broken. Are you sure the elapsed time validation isn't needed anymore?

Comment (andresmgot) [reply public/app/plugins/datasource/testdata/runStreams.ts:107]: yep, this is just executing `pushNextEvent` using `speed` as the period

**Review by jackw — APPROVED** [review]
LGTM! 🚀

**Review by ryantxu — APPROVED** [review]
yup -- we can return to this if we need to test liveTimer more, but for the current goals I think this is the best approach

**Pull request merged**